README.markdown

```diff
@@ -296,15 +274,6 @@ func reticulateSplines(spline: [Double]) -> Bool {
 }
 ```
 
-For functions with long signatures, add line breaks at appropriate points and add an extra indent on subsequent lines:
```

Comment:
yeah, that's hard to read. i'd suggest renaming the method for something like that. i still like the colons to line up for the arguments, though, instead of a run-on statement on one line.